be/src/olap/merger.cpp

```diff
@@ -41,8 +41,9 @@ Status Merger::merge_rowsets(TabletSharedPtr tablet, ReaderType reader_type,
     reader_params.reader_type = reader_type;
     reader_params.rs_readers = src_rowset_readers;
     reader_params.version = dst_rowset_writer->version();
-
     reader_params.tablet_schema = cur_tablet_schema;
+    reader_params.use_page_cache = false; // shouldn't use page cache
```

Comment:
No need to add this field. We can just use `reader_type` to judge whether to use page cache.
And for compaction operation, the reader_type is NOT `READER_QUERY`

Reply:
Yes, I see. but this field is not a new one, here just make sure it was set properly. BTW, If we have option for page cache switch, I think it's better to treat the option with top priority.